diff -ur smbc-1.1.1/configure.in smbc-1.1.1.new/configure.in
--- smbc-1.1.1/configure.in	2005-02-11 17:28:05.000000000 +0000
+++ smbc-1.1.1.new/configure.in	2005-02-14 23:16:53.433422504 +0000
@@ -44,7 +44,7 @@
 AC_CHECK_HEADERS([errno.h])
 AC_CHECK_HEADERS([time.h])
 AC_CHECK_HEADERS([termios.h])
-AC_CHECK_HEADERS([curses.h])
+AC_CHECK_HEADERS([ncurses/curses.h])
 AC_CHECK_HEADERS([dirent.h])
 AC_CHECK_HEADERS([signal.h])
 AC_CHECK_HEADERS([stdio.h])
diff -ur smbc-1.1.1/src/fnet.h smbc-1.1.1.new/src/fnet.h
--- smbc-1.1.1/src/fnet.h	2005-02-14 18:34:50.000000000 +0000
+++ smbc-1.1.1.new/src/fnet.h	2005-02-14 23:17:12.584511096 +0000
@@ -9,7 +9,7 @@
 */
 // -------------------------------------------------------------------
 #define Rfree(ptr) free(ptr); ptr = NULL
-#include <curses.h>
+#include <ncurses/curses.h>
 
 int readline(register int fd, register char *ptr, register int maxlen);
 int writen(register int fd, register char *ptr, register int nbytes);
diff -ur smbc-1.1.1/src/smb-pack.c smbc-1.1.1.new/src/smb-pack.c
--- smbc-1.1.1/src/smb-pack.c	2005-02-08 13:09:13.000000000 +0000
+++ smbc-1.1.1.new/src/smb-pack.c	2005-02-14 23:17:34.908117392 +0000
@@ -18,7 +18,7 @@
 #endif /* HAVE_STRINGS_H */
 #endif /* not HAVE_STRING_H */
 
-#include <curses.h>
+#include <ncurses/curses.h>
 #include <libsmbclient.h>
 #include <unistd.h>
 #include <stdlib.h>
