diff -urN smbc-1.1.2.orig/configure.in smbc-1.1.2/configure.in
--- smbc-1.1.2.orig/configure.in	2005-02-21 14:39:57.000000000 +0100
+++ smbc-1.1.2/configure.in	2005-02-24 00:39:02.927958344 +0100
@@ -44,7 +44,7 @@
 AC_CHECK_HEADERS([errno.h])
 AC_CHECK_HEADERS([time.h])
 AC_CHECK_HEADERS([termios.h])
-AC_CHECK_HEADERS([curses.h])
+AC_CHECK_HEADERS([ncurses/curses.h])
 AC_CHECK_HEADERS([dirent.h])
 AC_CHECK_HEADERS([signal.h])
 AC_CHECK_HEADERS([stdio.h])
diff -urN smbc-1.1.2.orig/src/fnet.h smbc-1.1.2/src/fnet.h
--- smbc-1.1.2.orig/src/fnet.h	2005-02-22 11:13:15.000000000 +0100
+++ smbc-1.1.2/src/fnet.h	2005-02-24 00:39:43.827740632 +0100
@@ -11,7 +11,7 @@
 #define Rfree(ptr)    { free(ptr); ptr = NULL; } 
 #define Rdelwin(ptr)  { delwin(ptr); ptr = NULL; } 
 #define RdelwinE(ptr)  { if (ptr) werase(ptr); delwin(ptr); ptr = NULL; } 
-#include <curses.h>
+#include <ncurses/curses.h>
 
 int readline(register int fd, register char *ptr, register int maxlen);
 int writen(register int fd, register char *ptr, register int nbytes);
diff -urN smbc-1.1.2.orig/src/smb-pack.c smbc-1.1.2/src/smb-pack.c
--- smbc-1.1.2.orig/src/smb-pack.c	2005-02-21 14:40:18.000000000 +0100
+++ smbc-1.1.2/src/smb-pack.c	2005-02-24 00:40:10.481688616 +0100
@@ -18,7 +18,7 @@
 #endif /* HAVE_STRINGS_H */
 #endif /* not HAVE_STRING_H */
 
-#include <curses.h>
+#include <ncurses/curses.h>
 #include <libsmbclient.h>
 #include <unistd.h>
 #include <stdlib.h>
