diff -urN smbc-1.0.0.orig/configure.in smbc-1.0.0/configure.in
--- smbc-1.0.0.orig/configure.in	2004-09-29 13:50:05.000000000 +0000
+++ smbc-1.0.0/configure.in	2004-10-12 06:42:42.018512072 +0000
@@ -42,7 +42,7 @@
 AC_CHECK_HEADERS([errno.h])
 AC_CHECK_HEADERS([time.h])
 AC_CHECK_HEADERS([termios.h])
-AC_CHECK_HEADERS([curses.h])
+AC_CHECK_HEADERS([ncurses/curses.h])
 AC_CHECK_HEADERS([dirent.h])
 AC_CHECK_HEADERS([signal.h])
 AC_CHECK_HEADERS([stdio.h])
diff -urN smbc-1.0.0.orig/src/fnet.h smbc-1.0.0/src/fnet.h
--- smbc-1.0.0.orig/src/fnet.h	2004-09-06 16:41:22.000000000 +0000
+++ smbc-1.0.0/src/fnet.h	2004-10-12 06:40:50.930400032 +0000
@@ -10,7 +10,7 @@
   last update: 2004-08-08
 */
 // -------------------------------------------------------------------
-#include <curses.h>
+#include <ncurses/curses.h>
 
 int readline(register int fd, register char *ptr, register int maxlen);
 int writen(register int fd, register char *ptr, register int nbytes);
diff -urN smbc-1.0.0.orig/src/smb-pack.c smbc-1.0.0/src/smb-pack.c
--- smbc-1.0.0.orig/src/smb-pack.c	2004-09-21 07:57:46.000000000 +0000
+++ smbc-1.0.0/src/smb-pack.c	2004-10-12 06:41:24.440305752 +0000
@@ -20,7 +20,7 @@
 #endif /* HAVE_STRINGS_H */
 #endif /* not HAVE_STRING_H */
 
-#include <curses.h>
+#include <ncurses/curses.h>
 #include <libsmbclient.h>
 #include <unistd.h>
 #include <stdlib.h>
